template/src/navigation/AppNav.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
```

Comment:
this file was modified and renamed from RootNav, since there is no technical "root" nav anymore.

it previously contained a switchNavigator as the "root nav" which rendered either AppNav or GuestNav.

now the switchNavigator logic is handled in App.tsx, this file only needs to be concerned with AppNav

Reply:
Sweet!